ambry-api/src/main/java/com.github.ambry/server/StatsSnapshot.java

```diff
@@ -25,6 +29,7 @@
  * {@link StatsSnapshot}'s subMap will contain all the containers in the account that is mapped with. At the leaf level
  * {@link StatsSnapshot}'s subMap will be null.
  */
+@JsonPropertyOrder({"value", "subMap"})
 public class StatsSnapshot {
```

Comment:
Is there a unit test for serializing/deserializing `StatsSnapshot`? Will jackson make sure to not try to add `value` to the subMap when deserializing?

Reply:
Good point. I will try to add one unit test for `StatsSnapshot` for both serialization and deserialization.

Reply:
Unit test added